app/lib/bk/compat/parsers/circleci/orbs.rb

```diff
@@ -28,6 +19,144 @@ def translator(action, _config)
           ]
         end
       end
+
+      # Handles Docker installation logic
+      class DockerInstaller
```

Comment:
while not against it, I don't think that this warrants a whole class on its own as it is only instantiated once to manipulate the config in a very straight-forward way